components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/gatewayartifactsynchronizer/DBSaver.java

```diff
@@ -96,4 +101,4 @@ public String getName() {
 
         return APIConstants.GatewayArtifactSynchronizer.DB_SAVER_NAME;
     }
-}
+}
```

Comment:
new line in the EOF.

Reply:
https://github.com/wso2/carbon-apimgt/pull/9546/commits/97796414dcecd5806df827b070c91470386312ea